docs/spark/tutorials/ml-sentment-analysis.md

```diff
@@ -0,0 +1,159 @@
+
+[Download and install](https://marketplace.visualstudio.com/items?itemName=MLNET.07) ML.NET Model Builder (preview).
+
+## Build your machine learning model
+
+1. Open Visual Studio and create a new C# Console App (.NET Core). Name the project *MLSparkModel*.
+
+1. In **Solution Explorer**, right-click the *MLSparkModel* project. Then select **Add > Machine Learning**.
+
+1. From the ML.NET Model Builder, select the **Sentiment Analysis** scenario tile.
+
+1. On the **Add data** page, upload the *Yelp* reviews data set. 
+
+1. Choose *Sentiment* from the **Columns to Predict** dropdown.
+
+1. On the **Train** page, set the time to train to *60 seconds* and select **Start training**. Notice the status of your training under **Progress**.
+
+1. Once Model Builder is finished training, evaluate the training results. You can type phrases into the text box below **Try your model** and select **Predict** to see the output.
+
+1. Select **Code** and then select **Added Projects** to add the ML model code to the solution.
+
+1. Notice that two projects are added to your solutions: **MLSparkModelML.ConsoleApp** and **MLSparkModelML.Model**.
+
+1. Double-click on your *MLSpark* C# project and notice that the following project reference has been added.
+
+   ```xml
+   <ItemGroup>
+       <ProjectReference Include="..\MLSparkModelML.Model\MLSparkModelML.Model.csproj" />
+   </ItemGroup>
+   ```
+
+## Create a console app
+
+1. Right-click on **MLSparkModelML.Console** in Solution Explorer, and select **Manage NuGet Packages**.
+
+1. Search for **Microsoft.Spark** and install the package. **Microsoft.ML** was installed for you by Model Builder.
+
+### Create a SparkSession
+
+1. Open the *Program.cs* file for **MLSparkModelML.ConsoleApp**. This file was auto-generated by Model Builder. Delete the `using` statements, the contents of the Main() method, and section for sample data generation.
+
+1. Add the following additional `using` statements to the top of the *Program.cs*:
+
+   ```csharp
+   using System;
+   using System.Collections.Generic;
+   using Microsoft.ML;
+   using Microsoft.ML.Data;
+   using Microsoft.Spark.Sql;
+   ```
+
+1. Add the following code to your `Main` method to create a new `SparkSession`. The Spark Session is the entry point to programming Spark with the Dataset and DataFrame API.
+
+   ```csharp
+   SparkSession spark = SparkSession
+        .Builder()
+        .AppName(".NET for Apache Spark Sentiment Analysis")
+        .GetOrCreate();
+   ```
+
+   Calling the *spark* object created above allows you to access Spark and DataFrame functionality throughout your program.
+
+### Create a DataFrame and print to console
+
+Read in the Yelp review data from the *yelp.csv* file as a `DataFrame`. Include `header` and `inferSchema` options.
```

Comment:
Maybe we could add some info around why we include `header` and `inferSchema`:

```suggestion
Read in the Yelp review data from the *yelp.csv* file as a `DataFrame`. Include `header` and `inferSchema` options. This will indicate that *yelp.csv* has a header row with column labels, and it will automatically infer the data's schema, which is the name and type of data in each column.
```

Reply:
Also potentially add a statement about replacing `DATA_FILEPATH` with the path to yelp.csv on your machine?